### PR TITLE
Add tempest job for ironic operator

### PR DESCRIPTION
**Pull request by son-vyas**

### CODE REVIEW

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/37111/rehearse-37111-pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest/1651224429432147968/artifacts/ironic-operator-build-deploy-tempest/openstack-k8s-operators-tempest/build-log.txt

**Comment by hjensas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by hjensas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

File: ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh [hunk 1]
```diff
@@ -34,7 +34,9 @@ tempest init openshift
 pushd ~/tempest/openshift
 
 TEMPEST_CONF_OVERRIDES=${TEMPEST_CONF_OVERRIDES:-}
-discover-tempest-config --os-cloud ${OS_CLOUD} --debug --create identity.v3_endpoint_type public ${TEMPEST_CONF_OVERRIDES}
+
+discover-tempest-config --os-cloud ${OS_CLOUD} --debug --create \
```
Comment (cescgina) [inline ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh:38]: Is the removal of `TEMPEST_CONF_OVERRIDES` intentional? It'll probably break the manila tests, which required extra config options, does it mess with the ironic tests?

Comment (cescgina) [reply ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh:38]: thanks!

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
green run - https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/37111/rehearse-37111-pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest/1668643640022405120/artifacts/ironic-operator-build-deploy-tempest/openstack-k8s-operators-tempest/artifacts/stestr_results.html

**Comment by viroel** [comment]
/pj-rehearse ack

job is passing

**Pull request merged**